src/olympia/versions/models.py

```diff
@@ -693,6 +693,18 @@ def can_be_disabled_and_deleted(self):
         return True
 
 
+class VersionReviewerFlags(ModelBase):
+    version = models.OneToOneField(
+        Version, primary_key=True, on_delete=models.CASCADE
+    )
+    needs_human_review_by_mad = models.BooleanField(default=False)
+    pending_rejection = models.DateTimeField(default=None, null=True)
+
+    @property
+    def needs_human_review(self):
```

Comment:
not sure I understand the need for the property if we're recreating the entire model - can we just rename the field to `needs_human_review`?

Reply:
I'm not sure we really need it, I'll look into it.

Reply:
The property is likely not used yet but we don't want to rename the field because it needs to be specific. And we'll want to move the `needs_human_review` flag from `Version` to this model instead in the future.

Reply:
why does it need to be specific?  And if it does, why not just name the field that.  What's the use case for having a field available under two names?

Reply:
I think we can get rid of the property for now, assuming it isnt used (but I think it is not used).

My idea was to use a property to answer the question "does this version need human review?", which can be the case for different reasons, each reason represented by a different field. In some cases, we only want to know whether a version needs human review but in other cases, we want to know if it is caused by mad or another scanner or something else. We're not there yet, so we can remove the property I think

Reply:
a8c0d81e654e4bc65766b161ffd2cc9de90d397d